README.md

```diff
@@ -7,22 +7,47 @@ npm install vue-lazzy-image
 ## How use lib
 
 ```ecmascript 6
-import lazyload from 'vue-lazzy-image'
+import CoreLazyLoading from 'vue-lazzy-image'
 
 ...
 
-directives: {
-  lazyload,
+components: {
+  CoreLazyLoading,
 },
 ```
 
 ## Template
 
 ```ecmascript 6
-<img
-  v-lazyload
-  src="image_link"
-  class="class_name"
-  alt="alt_name"
-/>
+<CoreLazyLoading>
+  <img
+    src="image_link"
+    class="class_name"
+    alt="alt_name"
+  />
+</CoreLazyLoading>
+```
```

Comment:
you need to write code encoding. It means - you have to write at the beginning 'javascript' in order to add some colors.

```javascript
const friends = ['Jovid', 'Tahmuras', 'Ibod', 'Nozim'];
```

example is provided!